interactions/utils/utils.py

```diff
@@ -231,3 +246,64 @@ def disable_components(
             for _components in components:
                 for component in _components.components:
                     component.disabled = True
+
+
+def get_channel_history(
+    http: "HTTPClient",
+    channel: Union[int, str, "Snowflake", "Channel"],
+    start_at: Optional[Union[int, str, "Snowflake", "Message"]] = MISSING,
+    reverse: Optional[bool] = False,
+    check: Optional[Callable[["Message"], bool]] = None,
+    maximum: Optional[int] = inf,
+) -> AsyncHistoryIterator:
+    """
+    Gets the history of a channel.
+
+    :param http: The HTTPClient of the bot
+    :type http: HTTPClient
+    :param channel: The channel to get the history from
+    :type channel: Union[int, str, Snowflake, Channel]
+    :param start_at?: The message to begin getting the history from
+    :type start_at?: Optional[Union[int, str, Snowflake, Message]]
+    :param reverse?: Whether to only get newer message. Default False
+    :type reverse?: Optional[bool]
+    :param check?: A check to ignore specific messages
+    :type check?: Optional[Callable[[Message], bool]]
+    :param maximum?: A set maximum of messages to get before stopping the iteration
+    :type maximum?: Optional[int]
+
+    :return: An asynchronous iterator over the history of the channel
+    :rtype: AsyncHistoryIterator
+    """
+    return AsyncHistoryIterator(
```

Comment:
Would it be better to do a local import of the history iterator? If you do that, then it will keep it so that the `utils` submodule doesn't depend on any other parts of the library during initialization